k8s/service_controller/server/controller.go

```diff
@@ -360,11 +360,9 @@ func (c *Controller) getBroker(serviceID string) (*model.ServiceBroker, error) {
 }
 
 // This is what we get sent to us
-type CreateServiceInstanceRequest struct {
-	Name            string                 `json:"name"`
-	ServicePlanGUID string                 `json:"service_plan_guid"`
-	Parameters      map[string]interface{} `json:"parameters,omitempty"`
-}
+// TODO: Get rid of these and use the ones from:
```

Comment:
Yea, this goes to what I mentioned above.  if I'm remembering correctly, I think this struct is for the CC API.
https://apidocs.cloudfoundry.org/240/service_instances/creating_a_service_instance.html